src/components/StatusStrip/StatusStrip.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState, useEffect, ReactNode } from "react";
+
+import "./_status-strip.scss";
+
+type Status = {
+  label: string;
+  count: number;
+};
+
+type Props = {
+  statusList: {
+    model: Status[];
```

Comment:
I don't think that this data structure makes much sense anymore as it no longer is required to be in context of a modal.. maybe something simple like: `type Props = Status[]`